src/jaxspec/model/additive.py

```diff
@@ -389,3 +383,75 @@ def integrand(kT, e, tin, p):
         integral = integrate_interval(integrand)
 
         return norm * 2.78e-3 * (0.75 / p) / tin * jax.vmap(lambda e: integral(tout, tin, e, tin, p))(energy)
+
+
+class Agauss(AdditiveComponent):
+    r"""
+    A simple Gaussian line profile in wavelength space.
+    If the width is $$\leq 0$$ then it is treated as a delta function.
+    The Zagauss variant computes a redshifted Gaussian.
+
+    $$\mathcal{M}\left( \lambda \right) =
+    \frac{K}{\sigma \sqrt{2 \pi}} \exp\left(\frac{-(\lambda - \lambda_L)^2}{2 \sigma^2}\right)$$
+
+    ??? abstract "Parameters"
+        * $\lambda_L$ : Wavelength of the line in Angström $\left[\text{\AA}\right]$
+        * $\sigma$ : Width of the line width in Angström $\left[\text{\AA}\right]$
+        * $K$ : Normalization $\left[\frac{\text{photons}}{\text{cm}^2\text{s}}\right]$
+    """
+
+    def continuum(self, wavelength) -> (jax.Array, jax.Array):
+        line_wavelength = hk.get_parameter("Lambda_l", [], init=HaikuConstant(12.39842))
+        sigma = hk.get_parameter("sigma", [], init=HaikuConstant(0.001))
+        norm = hk.get_parameter("norm", [], init=HaikuConstant(1))
+
+        return norm * jsp.stats.norm.pdf(wavelength, loc=line_wavelength, scale=sigma)
+
+
+class Zagauss(AdditiveComponent):
+    r"""
```

Comment:
I think it is the opposite here : you should divide the continuum with (1+z) and multiply the energy by (1+z)

Reply:
and same comment as before about keeping the energy input and rather convert the parameters to their equivalent in energy space

Reply:
Changes added: energy as entry + continuum multiplied by (1+z) because wavelength as output + wavelength divided by (1+z)